apps/dav/lib/DAV/CustomPropertiesBackend.php

```diff
@@ -176,6 +176,21 @@ public function propFind($path, PropFind $propFind) {
 			}
 		}
 
+		// substr of addressbooks/ => path is inside the CardDAV component
+		// three '/' => this a addressbook (no addressbook-home nor contact object)
+		if (substr($path, 0, 13) === 'addressbooks/' && substr_count($path, '/') === 3) {
+			$allRequestedProps = $propFind->getRequestedProperties();
+			$customPropertiesForShares = [
+				'{DAV:}displayname',
+			];
+
+			foreach ($customPropertiesForShares as $customPropertyForShares) {
+				if (in_array($customPropertyForShares, $allRequestedProps)) {
```

Comment:
```suggestion
				if (in_array($customPropertyForShares, $allRequestedProps, true)) {
```

for strict comparison